app/com/gu/contentapi/sanity/SSLExpiryTest.scala

```diff
@@ -0,0 +1,63 @@
+package com.gu.contentapi.sanity
+
+import org.scalatest.{FlatSpec}
+import javax.net.ssl._
+import java.net.{ProtocolException, MalformedURLException, URL}
+import org.joda.time.{Days, DateTime}
+import java.io.IOException
+import sun.security.x509.X509CertImpl
+
+class SSLExpiryTest extends FlatSpec {
+
+  "SSL Certificates" should "be more than 30 days from expiry" taggedAs(LowPriorityTest) in {
+    handleException {
+      val hosts = Seq (
+        Config.host,
+        Config.hostPublicSecure,
+        Config.previewHost,
+        Config.writeHost,
+        Config.writePreviewHost
+      )
+
+      val secureHosts = for (host <- hosts) yield host.replaceAll("http://", "https://")
```

Comment:
Use map to do this more succinctly:
val secureHosts = hosts map {_.replaceAll("http://", "https://")}